docs/src/data/ignoredLinks.ts

```diff
@@ -21,4 +21,6 @@ export const IGNORED_LINKS = [
   'https://github.com/aws-amplify/amplify-ui-swift-liveness/issues/new/choose', // 302 amplify-ui-swift-liveness does not have issue templates yet
   'https://developer.apple.com/design/human-interface-guidelines/patterns/accessing-private-data/', // 401 Apple doesn't like bots maybe
   'https://twitter.com/AWSAmplify',
+  'https://tfhub.dev/tensorflow/tfjs-model/blazeface/1/default/1/model.json?tfjs-format=file', // 302 to a google cdn
+  'https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow/tfjs-backend-wasm/3.11.0/', // 404 this is the intentional path as the tfjs library will append the correct file name
```

Comment:
Would this still be needed if we treated the url as code, not a hyperlink in the docs?

Reply:
I believe so, will verify

Reply:
It's currently not a hyper link and it still gets parsed it seems